app/services/tsheets_service.rb

```diff
@@ -0,0 +1,29 @@
+class TsheetsService
+  def initialize
+    @adapter = TsheetsAdapter.new
+  end
+
+  def sync_users
+    unable_to_sync, synced = [], []
+    @tsheets_users = @adapter.find_users
+    @tsheets_users.each do |user|
```

Comment:
This is just meant to be a one off to sync existing users so they don't have to manually update everybody (moving forwards, new users would be linked in the app).

As it's a one off, I don't know if it's worth the bother of adding that kind of complexity. What are your thoughts, were you assuming this was an operation that might be run often?